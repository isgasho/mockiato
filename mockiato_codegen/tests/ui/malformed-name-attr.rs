#![feature(plugin)]
#![plugin(mockiato_codegen)]

#[mockable(name = 1)]
trait Foo {}

#[mockable(name(FooMock))]
trait Bar {}

#[mockable(name)]
trait Baz {}
