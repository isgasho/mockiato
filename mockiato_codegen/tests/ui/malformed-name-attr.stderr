error: #[mockable(name = "...") expects a string literal
 --> $DIR/malformed-name-attr.rs:4:12
  |
4 | #[mockable(name = 1)]
  |            ^^^^
  |
  = help: Example usage: #[mockable(name = "FooMock")]

error: #[mockable(name = "...") expects a string literal
 --> $DIR/malformed-name-attr.rs:7:12
  |
7 | #[mockable(name(FooMock))]
  |            ^^^^
  |
  = help: Example usage: #[mockable(name = "FooMock")]

error: #[mockable(name = "...") expects a string literal
  --> $DIR/malformed-name-attr.rs:10:12
   |
10 | #[mockable(name)]
   |            ^^^^
   |
   = help: Example usage: #[mockable(name = "FooMock")]

error: aborting due to 3 previous errors

